### PR TITLE
clean up  backend-mock a bit

### LINKED ISSUE #117: Restructure backend-mock with folders
Should have a folder structure to be more clean and the developer can find things faster.

Should have same structure as blitz_api

### PR DESCRIPTION
**Pull request by cstenglein**
closes #117 

Moved the routes to their own files.

More work will be needed later on, but current state is much better.

### CODE REVIEW

File: backend-mock/apps.js [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+  util.sendSSE("install", { id: "mempool" });
+  setTimeout(() => {
+    installApp();
+  }, 5000);
+  res.status(200).send();
+});
+
+router.post("/uninstall/:id", (req, res) => {
+  console.info("call to /api/v1/apps/uninstall for app", req.params.id);
+  // TODO
```
Comment (escapedcat) [inline backend-mock/apps.js:17]: What's the todo here?

Comment (cstenglein) [reply backend-mock/apps.js:17]: Doing some actions like the installing does. Not really important atm :)

Comment (escapedcat) [reply backend-mock/apps.js:17]: Wanna comment that?

**Review by escapedcat — APPROVED** [review]

File: backend-mock/apps.js (hunk 1 above)
Comment (cstenglein) [reply backend-mock/apps.js]: Sure, done! :+1:

**Pull request merged**